CHANGELOG.md

```diff
@@ -3,6 +3,99 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## Unreleased
+
+## Mainnet1B (agoric-upgrade-10)
+
+* Vaults
+    * Contract to mint IST from locked collateral
+    * Contract for auctions liquidating vaults with insufficient collateral
+    * A client for bidding in liquidation auctions
+    * Econ committee levers for adjusting vaults parameters
+* Oracle network
+    * Receives stream of asset price updates
+    * May trigger vault liquidation
```

Comment:
_potential fix_:
Not entirely accurate depending on what level of detail we're aiming for.

Oracles push new prices. The new prices are observed by Vaults which may then identify vaults to be liquidated. The Auction may then go liquidate.